modules/vactory_decoupled/src/JsonApiGenerator.php

```diff
@@ -130,6 +130,7 @@ public function fetch(array $config) {
     $response = $this->client->serialize($resource, $query_filters);
```

Comment:
@khalidbouhouch77
Nice! you exploited the existing [language:langcode] token 👍 
But developers could always forget the langcode filter in DF settings... 👎 
Let's try this approach: always add langcode filter to `$query_filters` whenever the concerned resource has the langcode field.